index.js

```diff
@@ -15,7 +15,7 @@ module.exports = (css, settings) => {
     encoding: 'utf8'
   })
 
-  if (result.stderr) {
+  if (result.status !== 0) {
     if (result.stderr.includes('Invalid PostCSS Plugin')) {
```

Comment:
I think here we can try to use `result.error` (iirc it should be an instance of `Error`)

Reply:
It's not Error, it's undefined.

After adding:
```diff
diff --git a/index.js b/index.js
index 415fd1f..1ba9284 100644
--- a/index.js
+++ b/index.js
@@ -15,6 +15,10 @@ module.exports = (css, settings) => {
     encoding: 'utf8'
   })
 
+  console.log('result.status', result.status)
+  console.log('result.signal', result.signal)
+  console.log('result.error', result.error)
+
   if (result.status !== 0) {
     if (result.stderr.includes('Invalid PostCSS Plugin')) {
       console.error('Next.js 9 default postcss support uses a non standard postcss config schema https://err.sh/next.js/postcss-shape, you must use the interoperable object-based format instead https://nextjs.org/docs/advanced-features/customizing-postcss-config')

```

this was the output of running the test suite:

```
➜  styled-jsx-plugin-postcss git:(fix/process-exit) yarn test
yarn run v1.22.5
$ mocha test.js


  styled-jsx-plugin-postcss
result.status 0
result.signal null
result.error undefined
    ✓ applies browser list and preset-env features (1327ms)
result.status 0
result.signal null
result.error undefined
    ✓ applies plugins (659ms)
result.status 0
result.signal null
result.error undefined
    ✓ works with placeholders (685ms)
result.status 0
result.signal null
result.error undefined
    ✓ works with @import (664ms)
result.status 0
result.signal null
result.error undefined
    ✓ works with quotes and other characters (669ms)
result.status 1
result.signal null
result.error undefined
    ✓ throws with invalid css (636ms)
result.status 1
result.signal null
result.error undefined
Next.js 9 default postcss support uses a non standard postcss config schema https://err.sh/next.js/postcss-shape, you must use the interoperable object-based format instead https://nextjs.org/docs/advanced-features/customizing-postcss-config
    ✓ throws with invalid config (131ms)


  7 passing (5s)

✨  Done in 6.07s.
```

Reply:
got it. we can fix it later if needed. Thanks again for the PR!